Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -982,6 +965,22 @@ - (void)applyReferralCode:(NSString *)code andCallback:(callbackWithParams)callb
 
 // PRIVATE CALLS
 
++ (Branch *)getBranchInstance:(BOOL)isLive {
+    if (!currInstance) {
+        NSString *branchKey = [BNCPreferenceHelper getBranchKey:isLive];
+        if (!branchKey || [branchKey isEqualToString:NO_STRING_VALUE]) {
+            NSLog(@"Branch Warning: Please enter your branch_key in the plist!");
+        }
+        NSString *appId = [BNCPreferenceHelper getAppKey];
```

Comment:
This won't quite work -- if you have one or the other, you shouldn't see a warning. This currently will show you a warning for each one you don't have (so, if you specified none, you'd see both warnings :smile:) We also probably don't want to tell them to set the old app key.